client/src/components/screens/PatientVariant/index.jsx

```diff
@@ -202,6 +204,37 @@ const clinVar = (variant, _gene) => {
   return cvcs || 0;
 };
 
+const cleanOmimValue = (value) => {
+  if (value == null) {
+    return '';
+  }
+  let output = value.trim();
+  if (output[0] === '{') {
+    output = output.substr(1);
+  }
+  if (output[output.length - 1] === '}') {
+    output = output.substr(0, output.length - 1);
+  }
+  return output;
+};
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const omimVar = (_variant, gene) => {
+  try {
+    const omims = gene.omim.map((omim) => {
+      let output = `- ${cleanOmimValue(omim.name)} MIM${omim.omim_id}`;
+      if (get(omim, 'inheritance', []).length > 0) {
+        output = `${output} (${omim.inheritance.join(', ')})`;
+      }
+      return output;
+    });
+
+    return omims.join('\n');
+  } catch (e) {
+    return '';
```

Comment:
Shouldn't we log it somewhere? (Something like sentry?) 'cause now if it fails, we would never know

Reply:
No, if the only way it could fail is when the information is not available, and in that case, we return an empty string